app/src/main/java/edu/ucsc/cmps115_spring2017/face2name/MainScreen.java

```diff
@@ -59,5 +66,43 @@ public void onPreviewReady() {
         mCameraPreview.startPreview();
     }
 
+
+
+    public boolean onTouchEvent(MotionEvent event) {
+        int l_x = mLayerView.getRectPoint("Left X");
+        int r_x = mLayerView.getRectPoint("Right X");
+        int u_y = mLayerView.getRectPoint("Upper Y");
+        int l_y = mLayerView.getRectPoint("Lower Y");
```

Comment:
Short variable names such as this are a hassle when reading code because they force the reader to refer back to their definition when you forget what they mean.